src/Tests/Messages/When_scrubbing_headers.cs

```diff
@@ -46,6 +46,41 @@ public class When_scrubbing_headers
                 .Equal(unallowed, (s1, s2) => string.Equals(s1, s2, StringComparison.OrdinalIgnoreCase));
         }
 
+        [Fact]
+        public void Should_always_allow_default_whitelist()
+        {
+            var whitelist = new string[] { "Only-This" };
+            var headers = new Dictionary<string, string> {
+                {
+                    "User-Agent", "something"
+                }
+            };
+
+            var result = HeaderScrubber.Scrub(headers, whitelist, new string[] { });
+
+            result.Should().Equal(headers);
+        }
+
+        [Fact]
+        public void Should_always_apply_default_blacklist()
+        {
+            var headers = new Dictionary<string, string> {
+                {
+                    "Authorization", "secret"
+                },
+                {
+                    "Cookie", "secret"
+                }
+            };
+
+            var result = HeaderScrubber.Scrub(headers, new string[] { }, new string[] { });
```

Comment:
I think it would be good to have a spec that shows that even when you pass a non-empty blacklist, the default blacklist is still applied. My first thought when looking at the interface is:

> Ok, there are some default whitelists and blacklists, so I guess I can override them

But that's not true :)

Reply:
Fixed!